### ISSUE TITLE
Screen reader will not announce pagination - current page

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#2986]
Screen reader will not announce current page because it is in a `<span>`:
```
<span class="page-link" aria-label="Current Page, Page 1" aria-current="true">1</span>
```
